### PR TITLE
Updated badass readme

### PR DESCRIPTION
**Pull request by nhance**
Updated badass readme

This is a longer body for the pull

### CODE REVIEW

**Comment by nhance** [comment]
lgtm this is indeed badass

**Comment by justincampbell** [comment]
turing.cool

**Comment by zachfeldman** [comment]
This is great

**Comment by Chrisell** [comment]
:-1:

**Comment by zachfeldman** [comment]
Sup philly rb

**Comment by zachfeldman** [comment]
Where's da pizza

File: README [hunk 1]
```diff
@@ -26,3 +26,6 @@ Your code will help keep our fish alive, or send him to an early grave.
 
 Some crap was changed here
 Some crap was added here
+
+NO JOKE THIS IS CRAZY
+Sneaky sneaky
```
Comment (nhance) [inline README]: Wtf is this??

**Comment by Chrisell** [comment]
LGTM

**Comment by zachfeldman** [comment]
RTFM

**Comment by zachfeldman** [comment]
LGTM

**Comment by justincampbell** [comment]
LGTM

**Comment by justincampbell** [comment]
wait no stahp

**Comment by Chrisell** [comment]
cat-jaw-drop-o

**Comment by zachfeldman** [comment]
aww

**Pull request closed without merging**